pages/calling_conventions.adoc

```diff
@@ -8,18 +8,18 @@ description: Examples of calling conventions on common platforms.
 
 A *calling convention* is the set of contracts that compiler-generated machine
 code respects and expects external functions to respect.
-The calling convention specifies, for example
+The calling convention specifies various aspects, such as:
```

Comment:
I think "for instance/example" sounds better than "various aspects" without "of ...", or better yet, omitting anything of this nature and making an exhaustive list

Reply:
The original phrasing was problematic, since the main body of the sentence was "the calling convention specifies", and I thought it was a bit awkward to have no object before the phrase "for example ...". One alternative would be "among other things", which makes it less awkward, since now the items of the list complete (as a object) the main phrase, and not a secondary phrase.